test/fixtures/keys/agent6.cnf

```diff
@@ -0,0 +1,18 @@
+[ req ]
+string_mask            = utf8only
+utf8                   = yes
+default_bits           = 1024
+days                   = 999
+distinguished_name     = req_distinguished_name
+attributes             = req_attributes
+prompt                 = no
+
+[ req_distinguished_name ]
+C                      = HU
+L                      = Budapest
+O                      = Tresorit
+CN                     = Ádám Lippai
```

Comment:
This is a copy of agent5.cnf which comes from the encoding fix in 11d57a535c21a64885808afd41612406ff854c69. Is it better to use another one?